packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.component.tsx

```diff
@@ -37,75 +35,74 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
   const { nonNavigationSelectPatientAction, showPatientSearch, hidePatientSearch, handleReturnToSearchList } =
     usePatientSearchContext();
 
-  const patientName = patient.person.personName.display;
-  const isDeceased = !!patient.person.deathDate;
+  const patientName =
```

Comment:
Do not construct the name to display manually. (May not be localized correctly)

```suggestion
  const patientName = getPatientName(patient);
```

Reply:
fixed